src/Microsoft.Extensions.ODataClient.Abstractions/ODataClientOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataClientOptions.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.ODataClient
+{
+    /// <summary>
+    /// An options class for configuring the default IODataClientFactory.
+    /// </summary>
+    public class ODataClientOptions
+    {
+        /// <summary>
+        /// The default logic name if no name is specified for the client when creating client from factory.
+        /// </summary>
+        public const string DefaultName = "";

```

Comment:
string.Empty

Reply:
I could not use string.Empty because that's static and not const, and for the DefaultName to show up in default parameter it has to be constant and could not be static, so it's on purpose here to use "".